docs/installation/server-config/index.rst

```diff
@@ -172,6 +158,18 @@ ksql.schema.registry.url
 
 The Schema Registry URL path to connect KSQL to.
 
+.. _ksql-service-id:
+
+^^^^^^^^^^^^^^^
+ksql.service.id
+^^^^^^^^^^^^^^^
+
+The service ID of the KSQL server. By default, the service ID of KSQL servers is ``ksql_``. KSQL servers that share the same
+``command`` topic belong to the same resource pool. Every internal topic has a fixed prefix of
```

Comment:
belong to the same resource pool -> belong to the same KSQL cluster

Reply:
I thought that we were using 'server pools' and not 'KSQL clusters' as a terminology. Did this change?

Most of the docs already refer to server pools.

Regardless, It would probably be more productive to fix this in a single pass audit after all the docs are merged.